{{cookiecutter.project_slug}}/frontend/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
+logs
```

Comment:
**suggestion** Having multiple `.gitignore` files works, but perhaps having only one file for the entire project is better, so when something is ignored it is more explicit (and easy) to identify why something was ignored. I would merge the content of this file with `{{cookiecutter.project_slug}}/.gitignore`, so backend and frontend files are maintained together.

Reply:
For me it is better to have them separate tbh. Because for me I usually only work on FE folder so having a gitigrone which shows the frontend related ignored files is more clear to me